cards.json

```diff
@@ -3,6 +3,10 @@
         "image": "assets/Images/accordion.png",
         "link": "Public/accordion.html"
     },
+ {
```

Comment:
Your json addition is wrong, why are you not following proper format of name, image, link.